### PR TITLE
NXP-27747: continues export

### PR DESCRIPTION
**Pull request by andreinechaev**

### CODE REVIEW

File: nuxeo-ai-model/src/main/java/org/nuxeo/ai/listeners/ContinuesExportListener.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import org.nuxeo.ecm.core.query.sql.SQLQueryParser;
+import org.nuxeo.ecm.core.query.sql.model.DateLiteral;
+import org.nuxeo.ecm.core.query.sql.model.IntegerLiteral;
+import org.nuxeo.ecm.core.query.sql.model.Predicate;
+import org.nuxeo.ecm.core.query.sql.model.Reference;
+import org.nuxeo.ecm.core.query.sql.model.SQLQuery;
+import org.nuxeo.ecm.core.query.sql.model.WhereClause;
+import org.nuxeo.runtime.api.Framework;
+
+public class ContinuesExportListener implements PostCommitEventListener {
```
Comment (PedroCardoso) [inline nuxeo-ai-model/src/main/java/org/nuxeo/ai/listeners/ContinuesExportListener.java]: Some documentation would be nice, and on key methods.

Comment (andreinechaev) [reply nuxeo-ai-model/src/main/java/org/nuxeo/ai/listeners/ContinuesExportListener.java]: true. need to add javadoc and readme.

File: README.md [hunk 2]
```diff
@@ -221,6 +221,24 @@ Nuxeo AI Core provides 3 Java modules:
...
 ```
+  * Continues Export - Once a week a Cron scheduler fires `startContinuesExport` event.
+  On such event the system will retrieve all AI Model defined under
+  configured project and perform evaluation of the current data to check if it needs to be uploaded.
+  ```xml
+  <extension target="org.nuxeo.ecm.core.scheduler.SchedulerService" point="schedule">
+    <schedule id="continues_export_default">
+      <eventId>startContinuesExport</eventId>
+      <!-- Every week at midnight -->
+      <cronExpression>0 0 0 1/7 * ?</cronExpression>
```
Comment (vpasquier) [inline README.md]: I would check that `every 7 days starting on the 1st, every month` is not starting too late if the addon is loaded after the 1st :D

File: nuxeo-ai-model/src/test/java/org/nuxeo/ai/listeners/ContinuesExportListenerTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    @Test
+    @Deploy("org.nuxeo.ai.ai-model:OSGI-INF/cloud-client-test.xml")
+    public void shouldLunchExportWithNoExceptions() {
+        EventBundle bundle = new EventBundleImpl();
+        EventContextImpl ctx = new EventContextImpl(session, session.getPrincipal());
+        bundle.push(new EventImpl("startContinuesExport", ctx));
+
+        ContinuesExportListener listener = new ContinuesExportListener();
+        listener.handleEvent(bundle);
```
Comment (vpasquier) [inline nuxeo-ai-model/src/test/java/org/nuxeo/ai/listeners/ContinuesExportListenerTest.java]: Is it testing something?

Comment (andreinechaev) [reply nuxeo-ai-model/src/test/java/org/nuxeo/ai/listeners/ContinuesExportListenerTest.java]: testing that there is no exceptions. Export itself already has been tested

**Pull request merged**